### DISCUSSION TITLE
스터디 - 15주차 목록

### DISCUSSION THREAD

**Discussion by vi2920va** [vi2920va/FE_interview#120] [General]
🏥 스터디 하실 분은 댓글 또는 아래의 내용을 수정해주세요.

## 시간표
2022.03.24(목) - 오후 9시
21:00 ~ 21:15 팀원들 PR 남긴 거 리뷰 시간
21:15 ~ 21:35 모의 면접,1:1, 1:다로 모의면접 진행(상대방이 공부한 것을 기준으로 2개 이상 질문 진행)
21:35 ~ 21:50 모의 면접 진행이 끝난 후, 토론 장소에 모여서 다른 분들에게 궁금했던 걸 토론을 진행
21:50 ~ 22:00 프론트엔드 모의 면접 질문 정리에 작성 후, 돌아가면서 소감 한마디 하고 스터디 종료 
  
## 남은 
- JavaScript - 프로토타입 상속

## 주현
- React - react router dom

## 두현
- Network - TCP IP 이어서

**Comment by NAMU1105** [comment]
남은

JavaScript - 프로토타입 상속

**Comment by userJu** [comment]
주현

React - react router dom
